gaffer-as-a-service/gaas-rest/src/test/java/uk/gov/gchq/gaffer/gaas/util/UnitTestConfig.java

```diff
@@ -70,10 +68,10 @@ public CustomObjectsApi customObjectsApi() {
         return new CustomObjectsApi(apiClient());
     }
 
-    @Bean
-    public AuthService authService() {
-        return new AuthService();
-    }
+//    @Bean
```

Comment:
Why are all these tests commented?

Reply:
These are all auth related tests. Rest no longer handles authentication internally so therefore, no longer has endpoints such as /auth or any sort of internal authentication mechanism. The authentication has been offloaded onto the sidecars therefore, the authentication is tested in the sidecar unit tests.

Reply:
Instead of them being commented out I have deleted them